templates/Translations/zh_CN/LC_MESSAGES/Alltube.po

```diff
@@ -0,0 +1,135 @@
+msgid ""
```

Comment:
If the gettext domain you are using is `AllTube` this file should probably be named `AllTube.po`.